front-end/src/api/DashboardApi.js

```diff
@@ -1,11 +1,42 @@
 import axios from 'axios';
 
+const url = process.env.NODE_ENV === 'development' ? 'http://localhost:3000' : 'https://githubconnector.herokuapp.com';
+
 export async function pullUserInfo(userID, token) {
-  const url = process.env.NODE_ENV === 'development' ? 'http://localhost:3000' : 'https://githubconnector.herokuapp.com';
+
   return axios.get(`${url}/users/${userID}`,{ headers:{
       'Accept': 'application/json',
       'Content-Type': 'application/json',
       'Authorization': 'Bearer '  + token }
   });
 }
 
+//Attempts to link Github Account
+export async function linkGithubAccount(userID, token, github_username) {
+  const config = {
+    headers:{
+      'Accept': 'application/json',
+      'Content-Type': 'application/json',
+      'Authorization': 'Bearer '  + token }
+  };
+  const payload = {
+    'github_username': github_username
```

Comment:
Another nitpick: I'd keep to the JavaScript standard and make variable names camelCase meaning:

```
{
  'github_username': githubUsername
}
```